05/piskvorky2.py

```diff
@@ -0,0 +1,50 @@
+def vyhodnot(pole):
+    # vraci vyhodnoceni pole
+    if "xxx" in pole:
+        return "x"
+    elif "ooo" in pole:
+        return "o"
+    elif "-" in pole:
+        return "-"
+    else:
+        return "!"
+
+def tah_hrace(pole):
+    # zkontroluje, ze tah hrace dava smysl
+    while True:
+        pozice = int(input("Zadej cislo policka:"))
+        if pozice < 0:
+            print("zaporna policka nejsou")
+        elif pozice > 19:
+            print("to je moc")
+        elif pole[pozice] != "-":
+            print("tam uz to nejde")
+        else:
+            break
+    return tah(pole, pozice, "o")
+
+def tah_pocitace(pole):
+    # definuje tah pocitace
+    import random
```

Comment:
importy je dávají všechny na začátek souboru :slightly_smiling_face: https://www.python.org/dev/peps/pep-0008/#imports
Je to tak hezčí a taky praktičtější, pro příště :slightly_smiling_face: